core/b2b/loader.tsx

```diff
@@ -4,11 +4,13 @@ import { auth } from '~/auth';
 
 import { ScriptDev } from './script-dev';
 import { ScriptProduction } from './script-production';
+import { ScriptProductionCustom } from './script-production-custom';
 
 const EnvironmentSchema = z.object({
   BIGCOMMERCE_STORE_HASH: z.string({ message: 'BIGCOMMERCE_STORE_HASH is required' }),
   BIGCOMMERCE_CHANNEL_ID: z.string({ message: 'BIGCOMMERCE_CHANNEL_ID is required' }),
   LOCAL_BUYER_PORTAL_HOST: z.string().url().optional(),
+  PROD_BUYER_PORTAL_HOST: z.string().url().optional(),
```

Comment:
I saw matt push for `HOST` for consistency - but I think the `PROD_BUYER_PORTAL_BASE_URL` makes more sense as it can be a sub directory.

`HOST` usually stands for hostname / domain / ip - which does not really indicate that you can have subdirectories.